standard/delegates.md

```diff
@@ -207,6 +207,43 @@ Once instantiated, a delegate instance always refers to the same invocation list
 
 > *Note*: Remember, when two delegates are combined, or one is removed from another, a new delegate results with its own invocation list; the invocation lists of the delegates combined or removed remain unchanged. *end note*
 
+> *Note*: Delegate combination and removal is known to fail with an exception in some implementations when the delegate types involved at runtime are different due to variant conversion ([§18.2.3.3](interfaces.md#18233-variance-conversion). The following example demonstrates a situation which may fail.
+> ```csharp
+> delegate void Action<in T>(T obj);
+>
+> class Test
+> {
+>    static void Main() {
+>       Action<string> stringAction = str => {};
+>       Action<object> objectAction = obj => {};
+>        
+>       // Valid due to implicit reference conversion of
+>       // objectAction to Action<string>, but may fail
+>       // at runtime.
+>       Action<string> combination = stringAction + objectAction;
+>    }
+> }
+> ```
+>
+> If combination or removal is required between  different but compatible delegate types, a *delegate_creation_expression* ([§12.7.11.6](expressions.md#127116-delegate-creation-expressions)) can be used to create a delegate with the correct runtime type. The following example demonstrates how this workaround may be applied to the preceding example.
```

Comment:
Note: I think "delegate creation expression" (rather than "*delegate_creation_expression*") would be better here, as we're referring to the semantic construct more than the syntax, but the rest of the section seems to use the latter. Thoughts welcome on this - and indeed whether it's even worth mentioning this workaround.